bool ON = True
bool OFF = False

#request
bool capture
float32 interval
---
bool result